src/console.c

```diff
@@ -3240,10 +3253,12 @@ void initConsole(Console* console, tic_mem* tic, FileSystem* fs, Config* config,
 				const char* first = argv[i];
 				const char* second = argv[i + 1];
 
+
 				if(cmdInjectCode(console, first, second)
 					|| cmdInjectSprites(console, first, second)
 					|| cmdInjectMap(console, first, second)
-					|| checkUIScale(console, first, second))
+					|| checkUIScale(console, first, second)
+					|| storeOptionalParams(console, first, second))
```

Comment:
We should update `argp` with `mask` if succeed handle the multiple values because we want to ignore "single value handling".

Reply:
Oh I think I know what it does now😄